### PR TITLE
READY: Add tooltip to votes column

### PR DESCRIPTION
**Pull request by ichorid**
This PR adds a detailed text tooltip to "Votes" column. In addition, it introduces a more general method of adding tooltips to columns.

### CODE REVIEW

**Comment by ichorid** [comment]
retest this please

File: TriblerGUI/widgets/tablecontentmodel.py [hunk 1]
```diff
@@ -138,12 +142,23 @@ def columnCount(self, parent=QModelIndex()):
     def flags(self, index):
         return self.column_flags[self.columns[index.column()]]
 
+    def filter_item_txt(self, txt_filter, index, show_default=True):
+        column = self.columns[index.column()]
+        data = self.data_items[index.row()][column] if column in self.data_items[index.row()] else u''
```
Comment (qstokkink) [inline TriblerGUI/widgets/tablecontentmodel.py]: Why not `self.data_items[index.row()].get(column, u'')`?

**Comment by ichorid** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Pull request merged**